### PR TITLE
Release/3.0.2

### PR DESCRIPTION
**Pull request by cklosowski**
https://github.com/awesomemotive/easy-digital-downloads/milestone/228?closed=1

| Issue # | Description |
|---------|-------------|
| #9319 | Duplicate % sign displayed when taxes is enabled |
| #9318 | Change the upgrade message if the orders table could not be created |
| #9316 | Notifications icon (number) is missing the styling on the front-end |
| #9307 | Customer verification link is not working in the admin |
| #9305 | JavaScript error when cancelling a transaction and PayPal is the only gateway |
| #9302 | Swedish Postal codes fail |
| #9295 | 3.0 - if any data should be migrated from EDD 2.x, the upgrade prompt should show |
| #9293 | Non Shop Manager roles cannot add notes to orders (and other objects) |
| #9288 | Bundle purchases don't show in the history-download short-code |
| #9280 | Searching the orders table no longer works as it used to |

### CODE REVIEW

**Comment by cklosowski** [comment]
@robincornett @zkawesome Here is the final release PR for EDD 3.0.2 if you want to have a look to make sure I didn't miss anything on version bumps.

File: readme.txt [hunk 1]
```diff
@@ -157,13 +157,13 @@ If Easy Digital Downloads doesn't quite fit your needs, take a look at some of o
...
 4. Insert purchase buttons for any download via the "Insert Download" button next the Upload Media buttons
-5. For detailed setup instructions, vist the official [Documentation](http://docs.easydigitaldownloads.com/?utm_source=docs&utm_medium=installation_tab&utm_content=documentation&utm_campaign=readme) page.
+5. For detailed setup instructions, vist the official [Documentation](https://easydigitaldownloads.com/docs/quickstart-guide/?utm_source=docs&utm_medium=installation_tab&utm_content=documentation&utm_campaign=readme) page.
 
 == Frequently Asked Questions ==
 
 = Where can I find complete documentation? =
 
-Full searchable docs for Easy Digital Downloads and all our extensions can be found at [http://docs.easydigitaldownloads.com/](http://docs.easydigitaldownloads.com/?utm_source=docs&utm_medium=faq_tab&utm_content=documentation&utm_campaign=readme)
+Full searchable docs for Easy Digital Downloads and all our extensions can be found at [http://easydigitaldownloads.com/docs](http://easydigitaldownloads.com/docs/?utm_source=docs&utm_medium=faq_tab&utm_content=documentation&utm_campaign=readme)
```
Comment (robincornett) [inline readme.txt:166]:
```suggestion
Full searchable docs for Easy Digital Downloads and all our extensions can be found at [https://easydigitaldownloads.com/docs](https://easydigitaldownloads.com/docs/?utm_source=docs&utm_medium=faq_tab&utm_content=documentation&utm_campaign=readme)
```

File: readme.txt [hunk 2]
```diff
@@ -239,6 +239,15 @@ Yes, through the use of our commercial addon called [Recurring Payments](https:/
 
 == Changelog ==
 
+= 3.0.2, July 26, 2022 =
+* Fix: Customer verification URL was not working in the admin.
+* Fix: When cancelling an order once at PayPal, the checkout page could produce a Javascript error.
+* Fix: Swedish postal codes would not pass validation.
+* Fix: Non-Shop Manager roles could no longer leave notes on orders.
+* Fix: Searching orders was not working in the admin.
+* Fix: Template: Download History shortcode was not providing access to download bundeled products.
```
Comment (robincornett) [inline readme.txt:248]:
```suggestion
* Fix: Template: Download History shortcode was not providing access to bundled products.
```

**Review by robincornett — CHANGES_REQUESTED** [review]

**Comment by cklosowski** [comment]
Changes accepted @robincornett

File: readme.txt [hunk 3]
```diff
@@ -117,7 +117,7 @@ The Easy Digital Downloads API makes it possible for developers to make customiz
 
 = Get help =
 
-Easy Digital Downloads is backed by top-notch technical support from our globally distributed full-time support team. We also have an [extensive documentation site available](https://docs.easydigitaldownloads.com/?utm_medium=readme&utm_source=wporg&utm_campaign=lite-plugin&utm_content=docs&utm_term=description). If you're looking for faster support via email, we encourage you to [purchase an Easy Digital Downloads pass](https://easydigitaldownloads.com/pricing/?utm_medium=readme&utm_source=wporg&utm_campaign=lite-plugin&utm_content=upgrade&utm_term=description) or premium extension.
+Easy Digital Downloads is backed by top-notch technical support from our globally distributed full-time support team. We also have an [extensive documentation site available](https://easydigitaldownloads.com//?utm_medium=readme&utm_source=wporg&utm_campaign=lite-plugin&utm_content=docs&utm_term=description). If you're looking for faster support via email, we encourage you to [purchase an Easy Digital Downloads pass](https://easydigitaldownloads.com/pricing/?utm_medium=readme&utm_source=wporg&utm_campaign=lite-plugin&utm_content=upgrade&utm_term=description) or premium extension.
```
Comment (robincornett) [inline readme.txt:120]:
```suggestion
Easy Digital Downloads is backed by top-notch technical support from our globally distributed full-time support team. We also have an [extensive documentation site available](https://easydigitaldownloads.com/?utm_medium=readme&utm_source=wporg&utm_campaign=lite-plugin&utm_content=docs&utm_term=description). If you're looking for faster support via email, we encourage you to [purchase an Easy Digital Downloads pass](https://easydigitaldownloads.com/pricing/?utm_medium=readme&utm_source=wporg&utm_campaign=lite-plugin&utm_content=upgrade&utm_term=description) or premium extension.
```

**Review by robincornett — APPROVED** [review]

File: readme.txt (hunk 3 above)
Comment (cklosowski) [reply readme.txt]: Fixing, as this needs to be to the `/docs` not homepage.

**Review by robincornett — APPROVED** [review]

**Pull request merged**